src/pre.l

```diff
@@ -928,9 +928,8 @@ WSopt [ \t\r]*
                                           BEGIN(FindDefineArgs);
                                         }
 <ArgCopyCComment>\n                     {
-                                          yyextra->defArgsStr+=' ';
+                                          yyextra->defArgsStr+=yytext;
                                           yyextra->yyLineNr++;
-                                          outputChar(yyscanner,'\n');
```

Comment:
Will this not cause the line numbers to get out of sync? (i.e. increasing `yyLineNr` without outputting a `\n`)